scripts/wrappers/wasm-opt

```diff
@@ -14,7 +14,6 @@ function logStatus {
 }
 
 if [[ -z "${WASMLABS_SKIP_WASM_OPT}" ]]; then
-   logStatus "wasm-opt (with args \"$@\") was intercepted; executing $BINARYEN_PATH/bin/wasm-opt"
```

Comment:
Why remove this one? I think it's good to have this output

Reply:
Yes, the reason is I was getting too many of those messages and generating noise while I was debugging other stuff. I'm going to restore it and later on we can discuss how to improve all the tracing output.